src/style.css

```diff
@@ -0,0 +1,153 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts\Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  font-weight: medium;
+  src: url(fonts\Roboto-Medium.ttf);
+}
+
+@font-face {
+  font-family: Roboto;
+  font-weight: bold;
+  src: url(fonts\Roboto-Bold.ttf);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.field {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+}
+
+.logo {
+  width: 160px;
+  height: 134px;
+  padding: 34px 19px 37px;
+}
+
+.field__text-name {
+  font-family: "Roboto", sans-serif;
+  font-weight: medium;
+  font-size: 12px;
+  line-height: 18px;
+  padding: 0 16px 4px;
```

Comment:
remove padding, use margin-bottom for all elements inside to make a distance between next element